scripts/settings.js

```diff
@@ -123,14 +123,14 @@ export default class Settings {
       Constants.MODULE_ID,
       Settings.STATUS_EFFECTS_SORT_ORDER,
       {
-        name: 'Status Effects Sort Order',
-        hint: 'This is how status effects are sorted in the token HUD. Requires a Foundry reload on change.',
+        name: game.i18n.localize("Settings.StatusEffectsSortOrder"),
+        hint: game.i18n.localize("Settings.StatusEffectsSortOrderHint"),
         scope: 'world',
         config: true,
-        default: 'none',
+        default: game.i18n.localize("Settings.StatusEffectsSortOrderNone"),

```

Comment:
Same here. And actually, it looks like there is a bug because default should have been `byOrderAdded`

Reply:
Will set it to byOrderAdded then :smile: